### PR TITLE
Unit Testing functions in `meta-analysis` sub-workflow

### PR DESCRIPTION
**Pull request by meetagrawal09**
## Description

This PR aims at adding unit tests for all the pivotal function in the `meta-analysis` sub-workflow.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [X] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] My name is in the list of CITATION.cff
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: modules/meta.analysis/tests/testthat/test.run.meta.analysis.R
Comment (dlebauer) [inline modules/meta.analysis/tests/testthat/test.run.meta.analysis.R:1]: note that the argument to `test_that` should be a description of the tests within. The `context` function can name the function or set of functions being tested.

Comment (infotroph) [reply modules/meta.analysis/tests/testthat/test.run.meta.analysis.R:1]: The testthat docs now discourage use of `context` and recommend encoding that info in the filename

File: modules/meta.analysis/tests/testthat/test.jagify.R
Comment (infotroph) [inline modules/meta.analysis/tests/testthat/test.jagify.R:29]: It's probably save to drop these lines:

1. You can get the same effect by passing `stringsAsFactors = FALSE` to the `data.frame` call
2. In R >= 4.0.0 (as all our supported versions are), stringsAsFactors is FALSE by default, so all of this is a no-op.

File: modules/meta.analysis/R/meta.analysis.R
Comment (meetagrawal09) [inline modules/meta.analysis/R/meta.analysis.R:9]: @infotroph, I think we should remove these outdated license blocks before a release. Should I remove all of them for the `meta.analysis` package and make a commit or doing it as a separate PR would be better ?

Comment (infotroph) [reply modules/meta.analysis/R/meta.analysis.R:9]: @meetagrawal09 It should be a separate PR that will merge into branch `release/v1.8.0` instead of into `develop`. We'll need to double-check that all code owners have signed off on the license change before we merge, but do go ahead and open the PR please.

File: modules/meta.analysis/tests/testthat/test.jagify.R
Comment (infotroph) [inline modules/meta.analysis/tests/testthat/test.jagify.R]:
```suggestion
```

**Review by infotroph — APPROVED** [review]

**Pull request merged**
